hearth/lib/hearth/middleware.rb

```diff
@@ -1,5 +1,21 @@
 # frozen_string_literal: true
 
+module Hearth
+  # @api private
+  module Middleware
+    # Helper methods for logging to middleware.
+    module Logging
```

Comment:
Why a submodule for this instead of just having it in the `Middleware` module?  

I guess, I could see a case where you are implementing a middleware and only want a subset of the modules utilities.

Reply:
I couldn't think of a reason for this to be "is a Middleware" when it's just a logging utility. It felt more correct to have a Middleware Logging module that you include to do logging. These can probably be refactored later.

Reply:
Yeah, that makes sense.